Tests/OpenTelemetrySdkTests/Trace/SpanBuilderSdkTests.swift

```diff
@@ -221,12 +220,11 @@ class SpanBuilderSdkTest: XCTestCase {
 
     func testParentCurrentSpan() {
```

Comment:
should any of these tests validate that the original span was restored on the context?

Reply:
Absolutely, just added a test to check it. Thanks a lot